src/aap_eda/api/views/team.py

```diff
@@ -50,6 +53,10 @@
                 TeamSerializer,
                 description="Return the new team.",
             ),
+            status.HTTP_403_FORBIDDEN: OpenApiResponse(
+                None,
+                description="Create should be done through the platform ingress.",  # noqa: E501
```

Comment:
@bzwei Wont this message be misleading when we return a 403 because of RBAC or other reasons. Should this be a more specific exception that says ExternallyManagedResourceException.

Reply:
@bzwei extend_schema has an option called [exclude](https://drf-spectacular.readthedocs.io/en/latest/drf_spectacular.html#drf_spectacular.utils.extend_schema) Can we use that to remove the end points from the swagger UI and OpenAPI spec. 
we could set
``` 
exclude=settings.DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED
```

Reply:
403 is per the Jira issue spec